cli/module_registry.go

```diff
@@ -706,15 +706,15 @@ func sameModels(a, b []ModuleComponent) bool {
 	return true
 }
 
-func sendModuleUploadRequests(ctx context.Context, stream apppb.AppService_UploadModuleFileClient, file *os.File, stdout io.Writer) error {
+func sendModuleUploadRequests(ctx context.Context, stream apppb.AppService_UploadModuleFileClient, file *os.File, stderr io.Writer) error {
```

Comment:
Justification: Most cli tools that I know of put progress bars / info on stderr instead of stdout.